### ISSUE TITLE
Widget is not updating

### ISSUE THREAD

**Issue by u3813** [FossifyOrg/Notes#68]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Notes/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Notes/issues) or [closed](https://github.com/FossifyOrg/Notes/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Notes/discussions) or [closed](https://github.com/FossifyOrg/Notes/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Notes/blob/master/CONTRIBUTING.md).

### Affected app version

1.1.0

### Affected Android/Custom ROM version

Android 14, stock One UI 6.1 (non-rooted)

### Affected device model

Samsung S21 FE (SM-G990B2/DS)

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Create a widget, add some text, save and close the app
2. Open the widget and add or remove some of the text
3. Sometimes the app won't update the text that was changed, instead displaying the old version

### Expected behavior

The widget's text is supposed to be updated when changing things.

### Actual behavior

The widget's text is not being updated after a non-reproducible interval. Force stopping the app or deleting the cache don't help either. Only a system reboot fixes the issue.

This was behaviour was previously reported here: https://github.com/SimpleMobileTools/Simple-Notes/issues/715

**Comment by Aga-C** [comment]
Please write an exact model of your device in Affected device model, and if you have a stock or custom Android in Affected Android/Custom ROM version.

**Comment by naveensingh** [comment]
Yes, that would explain it. Samsung has a habit of breaking apps in the background.

A FAQ will be added, closing this now.
